src/gui/controller.py

```diff
@@ -106,7 +106,7 @@ def add_city_case_window(
         label="", tag=tag, show=True, parent=parent, width=width, height=height
     )
     dpg.add_text(
-        "Case Studies where we have the island Amager in Copenhagen, Denmark as a danger zone, \n and a case study with the city of Ravenna in Italy as a danger zone",
+        "Case Studies: \n\nCase study 1: danger zone is the island of Amager in Copenhagen, Denmark. \nFew exits and therefore chance of congestion. \n\nCase study 2: danger zone is the city of Ravenna in Italy. \nRecently experienced intense, devastating downpour.\n",
         parent=tag,
```

Comment:
Just a suggestion to phrase it more like this

```md
Case Study X: City, Country
The danger zone encompasses...
```

You can also do string concatenation like this in Python. This is valid syntax to concatenate the 6 strings into 1 argument. Can help make the string easier to read.

```suggestion
        "Case Studies:\n\n"
        "Case Study 1: Copenhagen, Denmark\n"
        "The danger zone encompasses Amager, which has limited evacuation routes, "
        "creating a high risk of congestion during emergencies.\n\n"
        "Case Study 2: Ravenna, Italy\n"
        "This coastal city experienced devastating flooding in May 2023 due to extreme rainfall..",
        parent=tag,
        wrap=0,
```